### DISCUSSION TITLE
Trivial question - Is tqdm allowed?

### DISCUSSION THREAD

**Discussion by ggoggam** [seominjoon/kaist-ai605#8] [Q&A] [answered]
Although it was stated that only Python 3.7 and PyTorch will be used for the assignment, are we allowed to bring some util libraries such as tqdm?

**Comment by seominjoon** [comment, marked as answer]
Yes, of course. Don't worry about using these utility libraries.

**Reply by ggoggam** [reply]
Got it. Thank you!
